app/assets/stylesheets/pghkids.scss

```diff
@@ -0,0 +1,40 @@
+@import url('http://fonts.googleapis.com/css?family=Open+Sans:300,400,700,800,600');
+// from bower_components
+@import "breakpoint-sass/stylesheets/breakpoint";
+
+@import "library/reset";
+@import "library/grid";
+// @import "library/variables";
```

Comment:
The other skins have kept a variables file in a site specific file [https://github.com/pgharts/trusty-festivity-extension/blob/pghkids-styling/app/assets/stylesheets/traf-stylesheets/library/_variables.scss](like this). Is it worth replicating that pattern for consistency?